spec.html

```diff
@@ -7519,6 +7520,7 @@ <h1>%ThrowTypeError% ( )</h1>
         </emu-alg>
         <p>The value of the [[Extensible]] internal slot of a %ThrowTypeError% function is *false*.</p>
         <p>The `"length"` property of a %ThrowTypeError% function has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+        <p>The `name` property of a %ThrowTypeError% function has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
```

Comment:
Note that with the policy of #1302 we'll want to write
```
The `"name"` property
```
here and in a handful of newly-added lines below.